frontend/public/components/operator-lifecycle-manager/package-manifest.tsx

```diff
@@ -111,8 +111,8 @@ export const PackageManifestsPage: React.SFC<PackageManifestsPageProps> = (props
     resources={[
       {kind: referenceForModel(PackageManifestModel), isList: true, namespaced: true, prop: 'packageManifest', selector: {matchExpressions: [{key: visibilityLabel, operator: 'DoesNotExist'}, {key: OPERATOR_HUB_LABEL, operator: 'DoesNotExist'}]}},
       {kind: referenceForModel(CatalogSourceModel), isList: true, namespaced: true, prop: 'catalogSource'},
-      {kind: referenceForModel(SubscriptionModel), isList: true, prop: 'subscription'},
-      {kind: referenceForModel(OperatorGroupModel), isList: true, prop: 'operatorGroup'},
+      {kind: referenceForModel(SubscriptionModel), isList: true, namespaced: true, prop: 'subscription'},
+      {kind: referenceForModel(OperatorGroupModel), isList: true, namespaced: true, prop: 'operatorGroup'},
```

Comment:
@alecmerdler is it correct to list these resources only for the current namespace?

Reply:
Should be, we end up [filtering to the active namespace here](https://sourcegraph.com/github.com/openshift/console/-/blob/frontend/public/components/operator-lifecycle-manager/package-manifest.tsx#L83) anyway.